### PR TITLE
adding documentation for post-enacted? and put-enacted?

### PR DESCRIPTION
**Pull request by afsalthaj**

### CODE REVIEW

File: doc/decisions.md [hunk 1]
```diff
@@ -38,10 +38,12 @@ modified-since?             | Was the resource modified since the date given in
 moved-permanently?          | Was the resource moved permanently?                                                                                           | false                                             |
 moved-temporarily?          | Was the resource moved temporarily?                                                                                           | false                                             |
 multiple-representations?   | Should the response be 310 multiple representations?                                                                          | false                                             |
+post-enacted?               | Was the post request finally processed?                                                                                       | true                                              |
```
Comment (ordnungswidrig) [inline doc/decisions.md]: Please add a `since` marker for the next release, which will be `0.15`

Comment (ordnungswidrig) [inline doc/decisions.md]: See `processable?` for an example

Comment (afsalthaj) [inline doc/decisions.md]: Sure.. 👍  I will add that now

**Comment by afsalthaj** [comment]
@ordnungswidrig  done 👍

**Comment by ordnungswidrig** [comment]
Great. Will merge (and thus publish) as soon as the release is out.

**Pull request merged**